### PR TITLE
Autocopy

### PR DESCRIPTION
**Pull request by brodi1**
Solves #4 

I know it's possible not to use wl-copy.....but for now I hope it's ok 😄

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -7,18 +7,31 @@ int main(int argc, char** argv, char** envp) {
...
     int currentlyParsing = 0;
     for (int i = 1; i < argc; i++) {
         std::string arg = argv[i];
 
         if (currentlyParsing == 0) {
-            if (arg == "--format") {
+            if (arg == "--autocopy") {
+                g_pHyprpicker->m_bAutoCopy = true;
+
+                std::filesystem::path wlcopy_bin_path{ "/usr/bin/wl-copy" };
```
Comment (vaxerski) [inline src/main.cpp:19]: wl-copy doesnt need to be in /usr/bin. Why even check? we can mention that in the README

File: src/events/Events.cpp [hunk 2]
```diff
@@ -151,6 +157,12 @@ void Events::handlePointerButton(void *data, struct wl_pointer *wl_pointer, uint
         }
         case OUTPUT_RGB:
         {
+            if (g_pHyprpicker->m_bAutoCopy) {
+                std::stringstream rbg_sstr;
```
Comment (vaxerski) [inline src/events/Events.cpp:161]: naming convention

File: src/events/Events.cpp [hunk 3]
```diff
@@ -142,6 +142,12 @@ void Events::handlePointerButton(void *data, struct wl_pointer *wl_pointer, uint
                 return result;
             };
 
+            if (g_pHyprpicker->m_bAutoCopy) {
+                std::stringstream hex_sstr;
```
Comment (vaxerski) [inline src/events/Events.cpp:146]: naming convention

File: src/events/Events.cpp [hunk 4]
```diff
@@ -142,6 +142,12 @@ void Events::handlePointerButton(void *data, struct wl_pointer *wl_pointer, uint
...
+                hex_sstr << "\\#" << toHex(COL.r) << toHex(COL.g) << toHex(COL.b) << std::endl;
```
Comment (vaxerski) [inline src/events/Events.cpp:147]: why `\\#`??

File: src/clipboard/Clipboard.cpp [hunk 5]
```diff
@@ -0,0 +1,9 @@
+#include "Clipboard.hpp"
+#include "../includes.hpp"
+
+void Clipboard::copy(std::string str) {
+
+    std::stringstream copy_cmd;
+    copy_cmd << "/usr/bin/wl-copy " << str << std::endl;
```
Comment (vaxerski) [inline src/clipboard/Clipboard.cpp:7]: no. `execlp` exists no??

Comment (brodi1) [reply src/clipboard/Clipboard.cpp]: whoops sorry my bad....you're right 😅

File: src/events/Events.cpp (hunk 4 above)
Comment (brodi1) [reply src/events/Events.cpp]: idk for some reason the whole program got stuck while using only #

File: src/main.cpp (hunk 1 above)
Comment (brodi1) [reply src/main.cpp]: >  we can mention that in the README

done [71cf782](https://github.com/hyprwm/hyprpicker/pull/7/commits/71cf782e1d539cbc3a03f6b8e62687541cb7db1b)

**Pull request closed without merging**